### PR TITLE
Devctr 101 add detail metrics

### PR DESCRIPTION
**Pull request by PrinceNebulon**

### CODE REVIEW

File: app/components/analytics-query-metrics-detail.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	}),
+	query: "default",
+	queues: computed('queueService.queues', function() {
+		return this.get('queueService').get('queues');
+	}),
+	init: function() {
+
+		this._super(...arguments);
+
+		// this.set('availableDetailMetrics', this.get('analyticsValueService').getDetailMetrics(this.get('query')).slice());
```
Comment (PrinceNebulon) [inline app/components/analytics-query-metrics-detail.js:17]: Commented out code should be removed

File: app/components/analytics-query-metrics.js [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
 	query: "default",
+	queues: computed('queueService.queues', function() {
+		return this.get('queueService').get('queues');
+	}),
 	init: function() {
+
 		this._super(...arguments);
 
-		this.set('availableMetrics', this.get('analyticsValueService').getMetrics(this.get('query')));
+		// this.set('availableMetrics', this.get('analyticsValueService').getMetrics(this.get('query')).slice());
```
Comment (PrinceNebulon) [inline app/components/analytics-query-metrics.js:17]: Commented out code should be removed

**Comment by PrinceNebulon** [comment]
@WitLi07 content looks good, just a couple minor things to clean up and this can be merged.

File: app/models/analytics-value-service-query-fallbacks.js [hunk 3]
```diff
@@ -100,6 +100,8 @@ class Fallbacks {
             "oUserPresences",
             "oUserRoutingStatuses"
         ].sort()
+        this.detailMetrics = ["testValue"]
```
Comment (PrinceNebulon) [inline app/models/analytics-value-service-query-fallbacks.js:103]: Remove testing values, use actual current values for fallbacks.

File: app/models/analytics-value-service-query-fallbacks.js [hunk 4]
```diff
@@ -525,11 +528,21 @@ class Fallbacks {
                 "oOnQueueUsers",
                 "oUserPresences",
                 "oUserRoutingStatuses",
-                "oWaiting"
+                "oWaiting123"
```
Comment (PrinceNebulon) [inline app/models/analytics-value-service-query-fallbacks.js:531]: Remove testing values, use actual current values for fallbacks.

File: app/models/analytics-value-service-query-fallbacks.js [hunk 5]
```diff
@@ -525,11 +528,21 @@ class Fallbacks {
...
+            ],
+            detailMetrics: [
+                "oActiveUsers",
+                "oInteracting",
+                "oMemberUsers",
+                "oOffQueueUsers",
+                "oOnQueueUsers",
+                "oUserPresences",
+                "oUserRoutingStatuses",
+                "oWaiting123"
```
Comment (PrinceNebulon) [inline app/models/analytics-value-service-query-fallbacks.js:545]: Remove testing values, use actual current values for fallbacks.

File: app/models/analytics-value-service-query-fallbacks.js [hunk 6]
```diff
@@ -557,7 +570,8 @@ class Fallbacks {
             ],
             dimensions: [
                 "userId",
-            ]
+            ],
+            detailMetrics: ["asf"]
```
Comment (PrinceNebulon) [inline app/models/analytics-value-service-query-fallbacks.js:574]: Remove testing values, use actual current values for fallbacks.

File: app/services/github-service.js [hunk 7]
```diff
@@ -4,32 +4,48 @@ import Ember from 'ember';
...
-			that._processSdkTags(tags);
-			if (!(that.get('isDestroyed') || that.get('isDestroying'))) {
-				that.set('jsSdkReleases', that._processSdkTags(tags));
-			}
-		});
+
+		try {
+			let that = this;
+			$.getJSON('https://api.github.com/repos/MyPureCloud/platform-client-sdk-javascript/tags').done(function(tags) {
+				// that._processSdkTags(tags);
```
Comment (PrinceNebulon) [inline app/services/github-service.js:38]: Remove commented out code if it's not needed

File: tests/integration/components/flow-aggregate-query-test.js [hunk 8]
```diff
@@ -12,14 +12,14 @@ test('it renders', function(assert) {
...
-  assert.equal(this.$().text().trim(), '');
+  assert.notEqual(this.$().text().trim(), '');
 
   // Template block usage:
-  this.render(hbs`
-    {{#flow-aggregate-query}}
-      template block text
-    {{/flow-aggregate-query}}
-  `);
+  // this.render(hbs`
```
Comment (PrinceNebulon) [inline tests/integration/components/flow-aggregate-query-test.js:18]: Remove commented out code

File: tests/unit/routes/api-explorer-test.js [hunk 9]
```diff
@@ -15,14 +15,15 @@ const purecloudStub = Ember.Service.extend({
...
+
       }
 });
 
-test('it exists', function(assert) {
-  let route = this.subject();
-
-  assert.ok(route.model().indexOf(`access_token=${fakeToken}`) > -1);
-});
+// test('it exists', function(assert) {
```
Comment (PrinceNebulon) [inline tests/unit/routes/api-explorer-test.js:26]: remove commented out code

File: tests/integration/components/flow-aggregate-query-test.js (hunk 8 above)
Comment (WitLi07) [reply tests/integration/components/flow-aggregate-query-test.js:18]: It was commented out originally when I cloned it. You want me to just remove it or leave it like that?

**Pull request merged**